playground/assets/__tests__/assets.spec.ts

```diff
@@ -364,3 +364,11 @@ test('relative path in html asset', async () => {
   expect(await page.textContent('.relative-js')).toMatch('hello')
   expect(await getColor('.relative-css')).toMatch('red')
 })
+
+test('url to public folder', async () => {
+  expect(await getBg('.style-public-assets')).toContain(iconMatch)
+  // TODO: To investigate why `await getBg('.inline-style-public') === "url("http://localhost:5173/icon.png")"`
+  // It supposes to be `url("http://localhost:5173/foo/icon.png")`
+  // (I built the playground to verify)
+  expect(await getBg('.inline-style-public')).toContain(iconMatch)
```

Comment:
This test is failing because during dev `await getBg('.inline-style-public')` is `"url("http://localhost:5173/icon.png")"`.
Vite's test runs with dev and then runs with build+preview.

I think there is a different bug here. `getBg('.inline-style-public')` should return `url("http://localhost:5173/foo/icon.png")` and not `url("http://localhost:5173/icon.png")"` during dev too. (because `src="/raw.js"`(at `assets/index.html` line 18) is rewrote to `src="/foo/raw.js"` during  dev.

Reply:
Yeah. I guess there is a bug in Vitest configuration here but I'm not familiar so I haven't catched it yet. I will investigate more. But if you spot the bug, please let me know, or just push the change directly to my branch. Much appreciated.

Reply:
Thank for your help. I will separate it to a different test case then skip it for now. I will push it soon.

Reply:
I just re-ran the code again in DEV mode and BUILD mode. I confirm that the `url()` in inline style is missing `base` url.

**Dev mode**
Missing `/foo`
![SCR-20220720-ukj](https://user-images.githubusercontent.com/8603085/180015630-244c07e6-65d2-4480-85fe-9b09b9c68768.png)

This is what I get when I visit `http://127.0.0.1:5173/icon.png` directly.
<img width="683" alt="image" src="https://user-images.githubusercontent.com/8603085/180017386-69f1bb2f-91b3-4c6c-ad8e-a850e715babf.png">

But I don't understand why it works fine a CSS rule
![SCR-20220720-urv](https://user-images.githubusercontent.com/8603085/180017656-ff0ecdaa-15fe-417c-b7e5-9b4f75feb9bd.png)

**Build mode**
Have `/foo`
![SCR-20220720-nbz](https://user-images.githubusercontent.com/8603085/180015720-b0d8b5a7-2061-4297-ba11-fde9f58f9f20.png)

So I guess we just discovered another bug.

I also push the commit to skip the failed test for DEV mode.